setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = quickmpc
-version = 0.1
+version = 0.1.2
```

Comment:
In pypi it is 0.0.1, so is it 0.0.2?

Reply:
Currently numbered 0.1.2 as the next version is 0.1.1.